font-codegen/src/fields.rs

```diff
@@ -1447,6 +1521,14 @@ fn width_for_offset(offset: &syn::Ident) -> Option<syn::Ident> {
     }
 }
 
+/// turn a syn path like 'std :: hmm :: Thing' into "Thing"
+fn stringify_path(path: &syn::Path) -> String {
```

Comment:
I would have expected stringify path to return the whole path as a string but this seemingly doesn't do that at all. I also don't love the parsing of token stream to string.

Wdyt about the alternative proposed in https://github.com/googlefonts/fontations/pull/946?

Reply:
so this is very narrowly scoped, and is specifically about getting a shorter name that we can use in certain diagnostic messages, for which we need a string; `syn::Path` does not implement `Display`.